product_docs/docs/edbcloud/beta/getting_started/01a_managing_quotas.mdx

```diff
@@ -0,0 +1,187 @@
+---
+title: Managing service limits and constraints in Azure
+
+---
+
+Your Azure subscription must meet certain quota (service limit) requirements to work with EDB Cloud. By default, Azure sets a very low limit on the number of VMs, vCPUs (cores), and public IP addresses permitted for a particular region in your subscription.
```

Comment:
>  Your Azure subscription must meet certain quota (service limit) requirements

Again, I think we should get away from "quotas" and start talking about Azure configurations.

Reply:
Made changes to remove the usage of qotas.